test/controllers/orders_controller_test.rb

```diff
@@ -0,0 +1,19 @@
+require 'test_helper'
+
+class OrdersControllerTest < ActionDispatch::IntegrationTest
+
+def new
+  @order = Order.new
+  authorize @order
+  @meal = Meal.find(params[:meal_id])
+
+end
+
+def create
+  @order = Order.new(order_params)
```

Comment:
You are using the order_params, but that means you have created a method with the same name. In kitt you can find it as Strong Params